src/components/DatePicker.vue

```diff
@@ -256,15 +256,15 @@ export default {
     ...Helpers,
 
     handleWindowResize() {
-      let screenSizeInEm = window.innerWidth / parseFloat(getComputedStyle(document.querySelector('body'))['font-size']);
+      // let screenSizeInEm = window.innerWidth / parseFloat(getComputedStyle(document.querySelector('body'))['font-size']);
```

Comment:
we should probably remove this comment if we wont use the em calculation anymore :)

Reply:
Oops! Thanks!